operator/pkg/manifest/shared.go

```diff
@@ -196,6 +196,9 @@ func GenIOPFromProfile(profileOrPath, fileOverlayYAML string, setFlags []string,
 		return "", nil, err
 	}
 
+	// resolve possible user setting conflicts
+	resolveConflictSettings(finalIOP, l)
```

Comment:
nit: more idiomatic to write this as `finalIOP = resolveConflictSettings(finalIOP, l)` to make it clear finalIOP is being mutated.